package-lock.json

```diff
@@ -6201,8 +6201,8 @@
       }
```

Comment:
is this an intentional change to package-lock.json?

Reply:
yeah, i removed it and force pushed